channels/rdpdr/server/rdpdr_main.c

```diff
@@ -1883,11 +1883,10 @@ static UINT rdpdr_server_drive_delete_directory_callback1(RdpdrServerContext* co
 static UINT rdpdr_server_drive_delete_directory(RdpdrServerContext* context, void* callbackData,
                                                 UINT32 deviceId, const char* path)
 {
-	RDPDR_IRP* irp;
+	RDPDR_IRP* irp = rdpdr_server_irp_new();
 	WINPR_ASSERT(context);
```

Comment:
Good catch, so surprised that we have not been flooded by assertion breaks before